packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -472,8 +472,7 @@ function FontCollection( { slug } ) {
 							className="font-library-modal__footer"
 						>
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> typogrpahy --> open font library modal and install fonts tab.

| Before | After |
| :-------- | :------- |
| <img width="404" alt="install-fonts-before" src="https://github.com/user-attachments/assets/9316e174-7413-4205-a9a4-ab3f08e51b67"> | <img width="404" alt="install-fonts-after" src="https://github.com/user-attachments/assets/e3bd82d3-7f3a-400a-8c8a-87d7dd253cbf"> |